src/borg/testsuite/logger.py

```diff
@@ -1,12 +1,27 @@
 import logging
 from io import StringIO
+import os
 
 import pytest
 
 from ..logger import find_parent_module, create_logger, setup_logging
+from .. import logger as _logger
 logger = create_logger()
 
 
+def reset_logging():
+    global logger
+    import importlib
+    # otherwise we'd need to reset the root logger and delete all other loggerss
+    importlib.reload(logging)
```

Comment:
iirc, reload() has all sorts of issues. does it really work here?

typo: loggerss

Reply:
In the meantime I came to the conclusion that re-setting the logging config shouldn't be our method of choice. See my argumentation below...